contrib/python-bindings/source/triangulation_wrapper.cc

```diff
@@ -1435,31 +1520,31 @@ namespace python
   void
   TriangulationWrapper::save(const std::string &filename) const
   {
-    std::ofstream                 ofs(filename);
-    boost::archive::text_oarchive oa(ofs);
+    std::ofstream                   ofs(filename);
+    boost::archive::binary_oarchive oa(ofs);
```

Comment:
Any reason for this change?

Reply:
Yes. The text format is slower and occupies much more space on the disk. This is especially apparent in 3-D (I am using this already for production runs...).